apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -82,7 +82,11 @@ Page {
 
                 onCurrentItemChanged: {
```

Comment:
May I propose to actually move this to the delegate itself.

Feel free to adapt from here:
https://github.com/sailfishos/sailfish-browser/commit/6d7e1fc4eadbc7b41255d8ad625ef12056b354da#diff-c27387fb9a98ac07eb836e3e56d5275ebc64fb1e685ab856f63946d4303e1c08R85

There are already roles regarding item being installed or not. Also url role is there.

So MenuItem click handled could be turned into somehting like:

```
onClicked: {
    if (title !== searchEngineConfig.value) {
        if (installed) {
            searchEngineConfig.value = title
        } else {
            searchFetcher.fetch(url)
        }
    }
}

```
No need for isInstalled and url invokables.

Reply:
Had problems using installed property here, it doesn't change until page is reloaded. Might be a problem with my dataChanged signal.